day_two_guide/topics/proc_backing-up-master.adoc

```diff
@@ -40,6 +40,8 @@ Because the masters are also nodes, back up the entire `/etc/origin` directory.
 You must perform the following steps on each master node.
 ====
 
+. Create a backup of the pod definitions, located xref:../day_two_guide/topics/proc_backing-up-etcd.adoc[here].
```

Comment:
no xrefs in modules

Reply:
@kalexand-rh it is for 3.11 so it should be ok to have xrefs in modules.

Reply:
@sheriff-rh I believe the issue is that you are linking to a module, and not to the assembly. The module doesn't exist by itself so the build is failing. 

You probably need to replace this with:

```suggestion
. Create a backup of the pod definitions, located xref:../day_two_guide/environment_backup.adoc#backing-up-etcd_environment-backup[here].
```
Trying the new suggested feature to see if you can simply accept the change without needing to re-push. :)